src/main/resources/lib/unsplash/types.ts

```diff
@@ -0,0 +1,33 @@
+export interface UnsplashImageMetadata {
+  id: string;
+  description: string;
+  alt_description: string;
+  user: {
+    name: string;
+  };
+  urls: {
+    thumb: string;
+    raw: string;
+  };
+}
+
+export interface SearchPhotosParams {
+  query: string;
+  start: number;
+  count: number;
+}
+
+export interface UnsplashResponseBody {
+  total: number;
+  total_pages: number;
+  results: UnsplashImageMetadata[];
+}
+
+export interface UnsplashContent {
+  data: UnsplashImageMetadata;
+}
+
+export interface UnsplashImageUrlParams {
```

Comment:
I would maybe have wanted some more fields here for more customization possibilities.

Reply:
Like what, for example? `alt`? Anything else?